lib/line-index/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "line-index"
+version = "0.1.0"
```

Comment:
I guess we might start with `0.1.0-pre.1` to be relaxed about shipping an obvious API bug and burning through too many versions too quicklly?